scipy/optimize/_qap.py

```diff
@@ -13,26 +13,24 @@ def quadratic_assignment(
     Solve the quadratic assignment problem.
 
     This function solves the Quadratic Assignment Problem (QAP) and the
-    Graph Matching Problem through an implementation of the Fast
-    Approximate QAP Algorithm (FAQ) (these two problems are the same up
-    to a sign change) [1]_.
+    Graph Matching Problem (GMP) using the Fast Approximate QAP Algorithm
+    (FAQ) [1]_.
 
-    Quadratic Assignment solves problems of the following form:
+    Quadratic assignment solves problems of the following form:
 
     .. math::
 
-        \min_P & \ {\ \text{trace}(APB^T P^T)}\\
+        \min_P & \ {\ \text{trace}(A^T P B P^T)}\\
```

Comment:
I think we should keep this consistent throughout, and it made sense to me to change here rather than below.